azure-pipelines.yml

```diff
@@ -92,7 +92,7 @@ jobs:
   - task: UsePythonVersion@0
     displayName: 'Use Python 3'
     inputs:
-      versionSpec: 3.x
+      versionSpec: 3.8
```

Comment:
I previously pinned Python version for Windows (#15691). Is 3.9 causing any problem on Linux?

Reply:
OK. I reverted it.